sdk/src/client/secret/ledger_nano.rs

```diff
@@ -329,7 +303,7 @@ impl SignTransactionEssence for LedgerSecretManager {
                                 }
                             } else {
                                 log::debug!("[LEDGER] unsupported output");
-                                return Err(Error::MiscError);
+                                return Err(Error::MiscError.into());
```

Comment:
it may be a bit uncommon but would I think worth mentioning at least is that you could just use `?` to get rid of the explicit `.into()` calls. Taking this line as an example this would reduce to:
```rust
Err(Error::MiscError)?
```

Reply:
I don't like this change 🤷‍♀️

Reply:
fair enough, but in some file you (then probably accidentally) did it yourself. Or maybe it wasn't you.